### PR TITLE
Update readme with folder structure

### PR DESCRIPTION
**Pull request by eddiesmo**
Added folder structure. 
@ilchemla please update it according to the folder names you've chosen.
How Do I add Itamar as a reviewer?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,39 @@
...
+|   ├── 480p
+|   └── originalSize 
+├── Annotations # folders containing binary annotations
+|   ├── 480p
+|   └── originalSize
+└── ImageSets
+    ├── Train.txt # contains a list of all the images for training
+    ├── trainval.txt # contains ALL the images for training and for validation    
+    └── val.txt # contains only the images for validation
+```
```
Comment (ilchemla) [inline README.md]: Maybe here we could be more specific and said that the ImageSets `*.txt` files include list of sequences and not a list of images.

File: README.md [hunk 2]
```diff
@@ -1,25 +1,39 @@
...
-With only hundreds of annotated videos, we believe every contribution has the potential to increase performance. 
+- There is a severe lack of data in the space of video object segmentation at the moment.
+With only hundreds of annotated videos, we believe every contribution has the potential to increase performance.
 In our internal (soon to be published) analysis we have shown that a joint training on the GyGO and DAVIS datasets yields improved inference results. [todo: confirm]
 - To promote a more open, sharing culture and encourage other researchers to join our efforts :) The DAVIS dataset and the research ecosystem that grew it have been massively useful for us. We hope the community will find our datasets useful as well.
 
+## Folder structure
+```
+.
+├── OriginalFrames # folders containing RGB videos
```
Comment (ilchemla) [inline README.md]: Rename `OriginalFrames` to `JPEGImages`

**Review by ilchemla — COMMENTED** [review]
Where are we going to publish the S3 link to the zip file?

**Comment by eddiesmo** [comment]
Done!

**Review by ilchemla — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,25 +1,42 @@
...
+On the one hand, the sequences are quite simple in that they are virtually devoid of occlusions, fast motions or many of the other complexity inducing attributes mentioned above.
 On the other hand, the objects in these videos vary wildly in their "semantic categories": toys, cloths, models and office stuff.
 The GyGO dataset specializes in smartphone captured videos and its frames are relatively sparse (the annotated video speed is ~5 fps).
 
 We release it publicly with two goals in mind:
-- There is a severe lack of data in the space of video object segmentation at the moment. 
-With only hundreds of annotated videos, we believe every contribution has the potential to increase performance. 
+- There is a severe lack of data in the space of video object segmentation at the moment.
+With only hundreds of annotated videos, we believe every contribution has the potential to increase performance.
 In our internal (soon to be published) analysis we have shown that a joint training on the GyGO and DAVIS datasets yields improved inference results. [todo: confirm]
```
Comment (ItamarF) [inline README.md]: Do you want to keep this [todo: confirm]?

File: README.md [hunk 4]
```diff
@@ -1,25 +1,42 @@
...
 - To promote a more open, sharing culture and encourage other researchers to join our efforts :) The DAVIS dataset and the research ecosystem that grew it have been massively useful for us. We hope the community will find our datasets useful as well.
 
+## Download the dataset
+https://s3.amazonaws.com/gygox-assets/gygo-dataset.tar.gz
```
Comment (ItamarF) [inline README.md]: Worth adding file size

**Review by ItamarF — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (ilchemla) [reply README.md]: The ZIP size is: `857.6MB`

**Comment by eddiesmo** [comment]
Added a couple more commits. Please approve

**Pull request merged**